workflow/standup.md

```diff
@@ -0,0 +1,6 @@
+# Standup
+
+Standup will be conducted everyday starting ``10:30am`` Monday to Friday. The venue for the virtual Standup will be conducted over ``Google Meet``. 
```

Comment:
How about we post the Meet link as well?

Reply:
Good idea. Let me add that.

Reply:
Recommendation added.